go.mod

```diff
@@ -30,4 +31,6 @@ require (
 	k8s.io/klog/v2 v2.9.0
 )
 
-go 1.13
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.5.4 // CVE-2021-32760
```

Comment:
I wonder if this replacement is needed?

Reply:
It appears to be needed, without this, all of the old containerd versions come back into `go.sum`